Framework/Kernel/CMakeLists.txt

```diff
@@ -518,6 +518,7 @@ target_link_libraries(Kernel
                       ${NETWORK_LIBRARIES}
                       span
                       ${HDF5_LIBRARIES}
+                      Json
```

Comment:
Do we need to link in `Json` whenever when ever we use Kernel? If so then we should move this to the `PUBLIC` section and then we can remove this from the `KernelTest` target